software-development/Improvements.md

```diff
@@ -0,0 +1,8 @@
+#Improvements
+
+#Engagement
+One of the possible issues in Open Source Software Development is engagement in the project. Commercial development has several mechanisms to ensure the stability of the project, like interface specifications, plans and habits regarding how things are done{{"mockus2002two" | cite}}. Communication can be a substitute for most of these mechanisms, although the "communication-only" approach is not scalable. The stability of the project engages the participants of the project and ensure a proper end product. Extrinsic motivation, like money, reputation and peer pressure, can also boost the engagement in the project.  
```

Comment:
stability of the project engages the participants **in** the project and ensure**s** a proper end product